cypress_test/integration_tests/desktop/writer/a11y_dialog_spec.js

```diff
@@ -10,10 +11,34 @@ describe(['tagdesktop'], 'Accessibility Writer Tests', function() {
         cy.getFrameWindow().then(function (win) {
             cy.spy(win.console, 'error').as('console:error');
 
+            const enableUICoverage = {
+                'Track': {
+                    'type': 'boolean',
+                    'value': true
+                }
+            };
+            win.app.map.sendUnoCommand('.uno:UICoverage', enableUICoverage);
+
             win.app.allDialogs.forEach(command => {
+                // these need a specific context
+                if (command == '.uno:ContourDialog' ||
+                    command == '.uno:TransformDialog') {
+                    cy.log(`Skipping missing-context dialog: ${command}`);
+                    return;
+                }
+                // not jsdialog enabled
+                if (command == '.uno:ChapterNumberingDialog') {
+                    cy.log(`Skipping non-jsdialog dialog: ${command}`);
+                    return;
+                }
+
                 cy.log(`Testing dialog: ${command}`);
-                win.app.map.sendUnoCommand(command);
+                cy.then(() => {
```

Comment:
Please add : `cy.cGet('.jsdialog-window').should('not.exist');` before invoking new one.
This will make us sure we didn't leave old one and it was successfully closed.